tests/algorithms/pytorch/test_newton_raphson.py

```diff
@@ -333,9 +330,6 @@ def test_pytorch_nr_algo_performance(
 ):
     perfs = network.clients[0].get_performances(compute_plan.key)
 
-    # This abs_ative error is due to the l2 regularization, mandatory to reach numerical stability.
-    # This fails on mac M1 pro with 1e-5
-    # TODO investigate
```

Comment:
has this been fixed with latest versions?

Reply:
I was assuming yes but you're right, I'll let the comment, sorry